web/src/components/tasks/anytime-action/tax-clearance-certificate/TaxClearanceStepTwo.tsx

```diff
@@ -35,33 +62,37 @@ export const TaxClearanceStepTwo = (props: Props): ReactElement => {
         <Heading level={2} styleVariant={"h3"}>
           {Config.taxClearanceCertificateStep2.requestingAgencySectionHeader}
         </Heading>
-        <Content className={"text-bold margin-bottom-05"}>
-          {Config.taxClearanceCertificateStep2.requestingAgencyLabel}
-        </Content>
-        <TaxClearanceCertificateAgencyDropdown />
+        <div id={`question-requestingAgencyId`} data-testid={"requestingAgency"}>
+          <TaxClearanceCertificateAgencyDropdown />
+        </div>
         <HorizontalLine />
         <div className="margin-top-3">
           <Heading level={2} styleVariant={"h3"}>
             {Config.taxClearanceCertificateStep2.businessInformationSectionHeader}
           </Heading>
         </div>
-        <div className="margin-bottom-2">
-          <FieldLabelProfile fieldName={"businessName"} />
-          <BusinessName inputWidth="full" />
+        <div className="margin-y-2">
+          <ProfileField fieldName={"businessName"} hideLine={true} hideTopSpace={true} isFullWidth={true}>
```

Comment:
Why did we replace FieldLabelProfile with ProfileField? The Business Name field renders the input.

Reply:
Same question for the other two instances? Previously we were using a label component and thats been replaced with the input component.